pegasus/sites.v3/code.org/public/diversity-2020.md.erb

```diff
@@ -1,164 +0,0 @@
----
-title: Diversity in K-12 computer science
```

Comment:
Was this page and `diversity-old` removed intentionally?

Reply:
@bethanyaconnor yep! Min said we could delete them since they're old versions of the `diversity` page.

Reply:
Great! Just wanted to double check